tests/test_voronoi_fps.py

```diff
@@ -0,0 +1,178 @@
+import unittest
+import numpy as np
+
+from sklearn.exceptions import NotFittedError
+from skcosmo.datasets import load_csd_1000r
+
+from skcosmo.feature_selection.voronoi_fps import VoronoiFPS
+from skcosmo.feature_selection import FPS
+
+
+class TestVoronoiFPS(unittest.TestCase):
+    def setUp(self):
+        self.X, self.y = load_csd_1000r(return_X_y=True)
+        # re-order features, as they were already FPS-ed
+        norms = np.linalg.norm(self.X, axis=0)
+        self.X = self.X[:, np.argsort(norms)]
+        self.idx = [
```

Comment:
Let's replace this with the explicit list with a call to FPS() so that we might be able to change datasets and still have the tests run